src/stories/material/Material.tsx

```diff
@@ -1,28 +1,35 @@
+import clsx from "clsx";
+
 export type MaterialProps = {
   url: string;
   animate: boolean;
   size: "small" | "medium" | "large";
+  tint: "20" | "40" | "60" | "80" | "120";
 };
 
-const placeholderBg = [
-  "bg-identity-tint-120",
-  "bg-identity-tint-80",
-  "bg-identity-tint-60",
-  "bg-identity-tint-40",
-  "bg-identity-tint-20",
-];
-
 export const Material = (props: MaterialProps) => {
-  // If Material does not have an img, show a random bg-color for each card.
-  const bg = placeholderBg[(Math.random() * placeholderBg.length) | 0];
+  const { size, animate, url } = props;
```

Comment:
Why not extract props here as well?:
```
  const { size, animate, url, props } = props;
```

Reply:
Good point. I was a bit confused over whether it will always be defined or not. But in either case, you're right.